p256/src/lib.rs

```diff
@@ -114,6 +114,11 @@ impl elliptic_curve::weierstrass::point::Compression for NistP256 {
     const COMPRESS_POINTS: bool = false;
 }
 
+impl elliptic_curve::weierstrass::point::Compaction for NistP256 {
+    /// NIST P-256 points are typically not compact.
+    const COMPACT_POINTS: bool = false;
```

Comment:
This is a new trait, and I don't see this constant being used anywhere. Can this trait be removed altogether?

Reply:
We were trying to match the weierstrass::Compression trait which also has no usage anywhere. For consistency?